### DISCUSSION TITLE
Access via phone

### DISCUSSION THREAD

**Discussion by marcus-cali** [nginx-proxy/nginx-proxy#2235] [Q&A]
I am a complete beginner when it comes to virtualization. Please forgive my probably easy-to-solve question: I have installed a Proxmox server. Several LXC containers with Turnkey WordPress are installed. Nginx Proxy Manager is used as a reverse proxy, running as a VM. The configuration is simple and works great. However, there is one exception: accessing the websites via iPhone. With the proxy turned on, I get a "closed connection". I have tried Safari, Edge, Chrome, and Firefox on my iPhone. The result is the same everywhere. When calling up the websites in a desktop browser, they are displayed correctly. Background information: I am using pfsense (Netgate 6100) as a firewall.

**Comment by buchdag** [comment]
Hi Marcus.

If you're referring to https://nginxproxymanager.com/, this project is completely unrelated to it.
